examples/job_crud.py

```diff
@@ -53,7 +55,16 @@ def create_job(api_instance, job):
     api_response = api_instance.create_namespaced_job(
         body=job,
         namespace="default")
-    print("Job created. status='%s'" % str(api_response.status))
+    # Need to wait for a second for the job status to update
+    sleep(1)
```

Comment:
api_response.status is the status immediately after creating 'job'. then controller will run to update the job status as the job is making progress. the time to take for job to run to completion is not determined, it could be short, could be long. the way to get status update is to Watch it.

Reply:
@yliaog Do i need to write a function to watch the status and print? If yes can you please elaborate it?

Reply:
https://github.com/kubernetes-client/python/blob/master/examples/pod_namespace_watch.py has an example for 'watch', or, you may periodically poll for status update.

Reply:
@yliaog Should we just print the status as soon as we get the JOB_NAME available in the watch code? like below
```v1 = client.CoreV1Api()
    w = watch.Watch()
    for event in w.stream(v1.list_pod_for_all_namespaces, timeout_seconds=10):
        if JOB_NAME in event['object'].metadata.name:
              api_response = api_instance.read_namespaced_job_status(
                name=JOB_NAME,
                namespace="default")
     return api_response.status```
     
Or should we wait for the Job to be in the RUNNING phase and then print the status?

Reply:
It would be great if you can print the watch events until the job succeeded/failed. This can be put into a separate function.

Reply:
@roycaihw @yliaog I have updated the code to Poll for status update until success/failure. Please check.